microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -255,3 +295,23 @@ def _ensure_driver_installed(
     reboot_tool.reboot_and_check_panic(log_path)
 
     _check_driver_installed(node)
+
+
+def _gpu_provision_check(min_pci_count: int, node: Node, log: Logger) -> None:
+    lspci = node.tools[Lspci]
+    start_stop = node.features[StartStop]
+
+    init_gpu = lspci.get_devices_by_type(constants.DEVICE_TYPE_GPU, force_run=True)
+    log.debug(f"Initial GPU count {len(init_gpu)}")
+    assert_that(len(init_gpu)).described_as(
+        "Number of GPU PCI device is not greater than 0"
+    ).is_greater_than_or_equal_to(min_pci_count)
+
+    start_stop.stop()
+    start_stop.start()
+
+    curr_gpu = lspci.get_devices_by_type(constants.DEVICE_TYPE_GPU, force_run=True)
+    log.debug(f"GPU count after reboot {len(curr_gpu)}")
+    assert_that(len(curr_gpu)).described_as(
+        "GPU PCI device count should be same after restart"
+    ).is_equal_to(len(init_gpu))
```

Comment:
make description clear, it should be stop, start, not reboot/restart

Reply:
updated